lms/djangoapps/discussion/rest_api/discussions_notifications.py

```diff
@@ -362,6 +373,12 @@ def send_reported_content_notification(self):
         ]}
         self._send_course_wide_notification("content_reported", audience_filters, context)
 
+    def _populate_context_with_ids_for_mobile(self, context, additional_context=False):
```

Comment:
Is `additional_context` needed as parameter here? It is not being used in the code

Reply:
Initially I added it to add more context but we don't need it now. Let me remove it.

Reply:
@muhammadadeeltajamul I have removed extra param.